get_hashsets.js

```diff
@@ -0,0 +1,47 @@
+"use strict";
+
+require("dotenv").load();
+
+const fs = require("fs");
+const path = require("path");
+const request = require("request");
+
+const HIBP_API_ROOT = process.env.HIBP_API_ROOT;
+const HIBP_API_TOKEN = process.env.HIBP_API_TOKEN || '';
+const HIBP_AUTH = 'Bearer ' + new Buffer(HIBP_API_TOKEN);
+
+
+function getBreachHashset(breach) {
+  if (breach.IsActive && breach.IsVerified && breach.DataClasses.indexOf("Email addresses") > -1) {
+    console.log("Active, verified breach with email addresses: " + breach.Name);
+    const url = HIBP_API_ROOT + '/enterprisesubscriber/hashset/' + breach.Name;
+    console.log("url: " + url);
+
+    const headers = {
+      'User-Agent': 'request',
```

Comment:
Instead of generic "request" User Agent, do you want to do use the package.json `name` and `version`?
For example:

```js
const pkg = require("./package.json");
// ...
const headers = {
  'User-Agent': `${pkg.name}@${pkg.version}`, // "blurts-server@1.0.0"
  'Authorization': HIBP_AUTH
};
```